src/solvers/dg/dg.jl

```diff
@@ -91,6 +91,52 @@ function Base.show(io::IO, ::MIME"text/plain", integral::VolumeIntegralLocalComp
 end
 
 
+# TODO: It would be really nice if we could convert other volume integrals to
+#       the flux form required here easily...
+# TODO: This needs a better name...
+struct VolumeIntegralFluxComparison{VolumeFlux_a, VolumeFlux_b} <: AbstractVolumeIntegral
+  volume_flux_a::VolumeFlux_a
+  volume_flux_b::VolumeFlux_b
+end
+
+function Base.show(io::IO, ::MIME"text/plain", integral::VolumeIntegralFluxComparison)
+  @nospecialize integral # reduce precompilation time
+
+  if get(io, :compact, false)
+    show(io, integral)
+  else
+    setup = [
+            "volume flux 1" => integral.volume_flux_a,
+            "volume flux 2" => integral.volume_flux_b
```

Comment:
I am not a big fan of alphabetical numbering, though in case of symmetric comparisons it looks less like there is an inherent ordering in the fluxes. I guess that naming is not yet final though, so maybe no no need to discuss this further now.

However, I'd prefer to make it consistent:
```suggestion
            "volume flux a" => integral.volume_flux_a,
            "volume flux b" => integral.volume_flux_b
```